pkg/build/builder/util/dockerfile/dockerfile_test.go

```diff
@@ -274,53 +274,6 @@ ENV PATH=/bin
 	}
 }
 
-// TestLastBaseImage tests calling LastBaseImage with multiple valid
-// combinations of input.
-func TestLastBaseImage(t *testing.T) {
```

Comment:
unopinionated except for the tests being deleted, what's the story there?

Reply:
> unopinionated except for the tests being deleted, what's the story there?

The functions they were testing no longer exist.  Dead code.

Reply:
i guess that's a good reason.